selfdrive/boardd/panda.cc

```diff
@@ -323,6 +323,7 @@ int Panda::can_receive(cereal::Event::Builder &event){
 
   // return if length is 0
   if (recv <= 0) {
+    event.initCan(0);
```

Comment:
if we're committed to this, we can just clip recv at 0 and get rid of the condition